proparse/src/main/java/org/prorefactor/proparse/ParserSupport.java

```diff
@@ -292,14 +293,14 @@ SymbolScope parseSuper(JPNode classNode, String qualSuperName)
     }
 
     try (BufferedReader ifstream = new BufferedReader(
-        new InputStreamReader(new FileInputStream(superFileName), doParse.getRefactorSession().getCharset()))) {
-      DoParse superDoParse = new DoParse(doParse.getRefactorSession(), superFileName, doParse);
+        new InputStreamReader(new FileInputStream(superFileName), session.getCharset()))) {
+      DoParse superDoParse = new DoParse(session, superFileName, /* FIXME doParse */ null);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1134)